tests/helpers/test_reports_helpers.py

```diff
@@ -104,97 +108,88 @@ def test_map_labware_to_location_labwhere_error(app, freezer, labwhere_samples_e
 
     try:
         with app.app_context():
-            map_labware_to_location(['test'])
+            map_labware_to_location(["test"])
     except ReportCreationError:
         raised_exception = True
 
     assert raised_exception
 
+
 def test_map_labware_to_location_dataframe_content(app, freezer, labwhere_samples_multiple):
     # mocks response from get_locations_from_labwhere() with labwhere_samples_multiple
     with app.app_context():
-        result = map_labware_to_location(['123', '456', '789']) # '789' returns no location, in the mock
+        result = map_labware_to_location(
+            ["123", "456", "789"]
+        )  # '789' returns no location, in the mock
 
-    expected_columns = ['plate_barcode', 'location_barcode']
-    expected_data = [
-        ['123', '4567'],
-        ['456', '1234'],
-        ['789', '']
-    ]
+    expected_columns = ["plate_barcode", "location_barcode"]
+    expected_data = [["123", "4567"], ["456", "1234"], ["789", ""]]
     assert result.columns.to_list() == expected_columns
     assert np.array_equal(result.to_numpy(), expected_data)
 
+
 def test_add_cherrypicked_column(app, freezer):
     # mocks response from get_cherrypicked_samples()
     existing_dataframe = pd.DataFrame(
-        [
-            ['MCM001', 'TEST'],
-            ['MCM002', 'TEST'],
-            ['MCM003', 'TEST']
-        ],
-        columns=['Root Sample ID', 'Lab ID']
+        [["MCM001", "123", "TEST"], ["MCM002", "123", "TEST"], ["MCM003", "123", "TEST"]],
+        columns=["Root Sample ID", "plate_barcode", "Lab ID"],
```

Comment:
Not sure what it does but is DataFrame `index=[0, 1, 2]` required? This it is used above

Reply:
No. Removed. Copied it from elsewhere.